pkgs/development/python-modules/hypothesis/default.nix

```diff
@@ -9,28 +9,19 @@ buildPythonPackage rec {
   # pytz fake_factory django numpy pytest
   # If you need these, you can just add them to your environment.
 
-  version = "3.45.2";
+  version = "3.66.2";
   pname = "hypothesis";
 
-  # Upstream prefers github tarballs
-  src = fetchFromGitHub {
```

Comment:
this should remain `fetchFromGitHub` because upstream doesn't include tests in their tarball

Reply:
They have completely reworked their testing infrastructure (there' no more setup.py even) so I haven't bothered reverse engineering it.